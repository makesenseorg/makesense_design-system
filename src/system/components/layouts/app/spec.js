import { shallowMount } from "@vue/test-utils";
import Comp from "./App.vue";

describe("App.vue", () => {
  it("matches snapshot", () => {
    const wrapper = shallowMount(Comp);
    expect(wrapper.element).toMatchSnapshot();
  });

  it("displays slots content", () => {
    const wrapper = shallowMount(Comp, {
      slots: {
        aside: "aside",
        default: "content",
        top: "top",
        bottom: "bottom",
      },
    });

    expect(wrapper.text()).toContain("aside");
    expect(wrapper.text()).toContain("content");
    expect(wrapper.text()).toContain("top");
    expect(wrapper.text()).toContain("bottom");
  });
});
