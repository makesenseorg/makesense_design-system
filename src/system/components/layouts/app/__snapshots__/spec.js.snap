// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue matches snapshot 1`] = `
<div
  class="app"
>
   
  <div
    class="app app--inner"
  >
    <!---->
      
    <!---->
  </div>
</div>
`;
